### ISSUE TITLE
Customize number of gifs

### ISSUE THREAD

**Issue by crabdancing** [wreiske/Rocket.Chat.App-Tenor#7]
It would be great if the user could customize the number of gifs they can scroll through -- like, instead of 10, they could set it to be more like 50.

**Comment by adocampo** [comment]
Agree, 10 to select is waaaaay short. I would like to set to unlimited or at least a wider range.
